core_backend/Makefile

```diff
@@ -9,7 +9,7 @@ tests: setup-test-containers run-tests teardown-test-containers
 # NB: `pytest-alembic` requires the DB engine to point an empty database. Thus, alembic
 # tests should be run first.
 run-tests:
-	@set -a && source ./tests/api/test.env && set +a && \
+	@set -a && . ./tests/api/test.env && set +a && \
```

Comment:
why did this change?

Reply:
OH so sorry, i by mistakenly comitted that change! thanks for pointing it out, so as am using WSL, as the main OS is still windows, the `source` command does not work for me so to run the makefiles i need to replace the `source` command